include/CppUTest/CppUTestConfig.h

```diff
@@ -199,6 +199,27 @@
 #define CPPUTEST_CHAR_BIT 8
 #endif
 
+/* Support for "long long" type */
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
My stupid mind thought you referred to when you try to use the checks on systems that don't support long long. I *thought* it would work also without #defining this, but now I see that isn't possible (yet)